### PR TITLE
22865 add heading to how to cancel message in appointment details

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Description
The PR add/modifies a heading to the how to cancel message box to draw attention to the information

## Testing done
Unit testing

## Screenshots
localhost_3001_health-care_schedule-view-va-appointments_appointments_va_a784f714fa1606e39dee11e3bdbb7901(iPhone X) (1)
---
localhost_3001_health-care_schedule-view-va-appointments_appointments_va_a784f714fa1606e39dee11e3bdbb7901(iPhone X) (2)

## Acceptance criteria
- [ ] All test should pass

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by LeKeve — APPROVED** [review]

File: src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx [hunk 1]
```diff
@@ -18,6 +18,9 @@ import ErrorMessage from '../../components/ErrorMessage';
 import { selectAppointmentById } from '../redux/selectors';
 import FullWidthLayout from '../../components/FullWidthLayout';
 import Breadcrumbs from '../../components/Breadcrumbs';
+import AlertBox, {
+  ALERT_TYPE,
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx:22]: Oh, interesting, I don't think I realized that was exported.

File: src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx [hunk 2]
```diff
@@ -145,10 +148,17 @@ function CommunityCareAppointmentDetailsPage({
...
       </div>
 
-      <div className="vads-u-margin-top--2 vaos-appts__block-label vads-u-background-color--primary-alt-lightest vads-u-padding--2p5">
-        Contact this provider if you need to reschedule or cancel your
+      <AlertBox
+        status={ALERT_TYPE.INFO}
+        className="vads-u-display--block"
+        backgroundOnly
+      >
+        <h2 className="vads-u-font-size--h3 vads-u-font-size--base vads-u-margin-top--0">
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx:156]: Was there a reason for putting the header here vs using the `headline` and `level` props?

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx [hunk 3]
```diff
@@ -135,6 +135,9 @@ export default function VideoVisitLocation({ header, appointment, facility }) {
           className="vads-u-display--block"
           backgroundOnly
         >
+          <h2 className="vads-u-font-size--h3 vads-u-font-size--base vads-u-margin-top--0">
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx:138]: Same question as above

**Review by jbalboni — APPROVED** [review]
This looks good, just have a nitpicky question that shouldn't block merging

File: src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx (hunk 2 above)
Comment (vbahinwillit) [reply src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx:156]: Nope. Just cut and paste from some other code. I will change to use props. The default level is 3 with is fine.

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx (hunk 3 above)
Comment (vbahinwillit) [reply src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx:138]: Nope. Just cut and paste from some other code. I will change to use props. The default level is 3 with is fine.

Comment (jbalboni) [reply src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx]: I think 2 was the intent there, but I could be wrong.

**Pull request merged**